test/jb_projects/challenge_handle.test.js

```diff
@@ -0,0 +1,164 @@
+import { expect } from 'chai';
+import { ethers } from 'hardhat';
+import { getTimestampFn } from '../helpers/utils'
+
+describe('JBProjects::challengeHandle(...)', function () {
+
```

Comment:
I've noticed this pattern where a space is left in between new blocks (in the tests as well) – for consistency let's try to align things the same.